src/ecalc/libraries/libecalc/common/tests/input/validation/snapshots/test_validation_json_schemas/test_json_schema_changed/schemas.json

```diff
@@ -32,20 +70,32 @@
                 },
                 "TIME_SERIES": {
                     "$ref": "https://test.ecalc.equinor.com/api/v1/schema-validation/time-series.json#properties/TIME_SERIES",
-                    "description": "Defines the inputs for time dependent variables, or \u201creservoir variables\u201d.\n\nhttps://test.ecalc.equinor.com/docs/docs/modelling/keywords/TIME_SERIES",
+                    "description": "Defines the inputs for time dependent variables, or 'reservoir variables'.\n\nhttps://test.ecalc.equinor.com/docs/docs/modelling/keywords/TIME_SERIES",
                     "title": "TIME_SERIES"
                 },
                 "VARIABLES": {
-                    "$ref": "https://test.ecalc.equinor.com/api/v1/schema-validation/definitions.json#definitions/VARIABLES"
+                    "additionalProperties": {
+                        "anyOf": [
+                            {
+                                "$ref": "#/definitions/SingleVariable"
+                            },
+                            {
+                                "additionalProperties": {
+                                    "$ref": "#/definitions/SingleVariable"
+                                },
+                                "type": "object"
+                            }
+                        ]
+                    },
+                    "description": "Defines variables used in an energy usage model by means of expressions or constants.\n\nhttps://test.ecalc.equinor.com/docs/docs/modelling/keywords/VARIABLES",
+                    "title": "VARIABLES",
+                    "type": "object"
                 }
             },
             "required": [
-                "INSTALLATIONS",
-                "TIME_SERIES",
-                "FACILITY_INPUTS",
-                "FUEL_TYPES"
+                "INSTALLATIONS"
```

Comment:
I think this is correct, great to see that schema and code is aligned :)

Reply:
Yes, we might need a conditional required here, not sure how to handle that. There are several combinations that is enough, i.e. Start, end and a direct consumer. 

I think FUEL_TYPES might be required though. Or do you have an example where it isn't?